cli/popper/commands/cmd_init.py

```diff
@@ -121,6 +121,7 @@ def initialize_repo(project_root):
         f.write('.cache\n')
         f.write('popper_logs\n')
         f.write('popper_status\n')
+        f.write('.*\n')
```

Comment:
This will include `.gitignore`, `.popper.yml`, etc., so basically all hidden files will be ignored. `.cache` is already ignored (line 121). I forgot to close that issue

Reply:
I have made some  changes here (for issues #145 and #152). 
I will be pushing them soon ( Will notify when done).